.github/workflows/zoltar-upload.yml

```diff
@@ -0,0 +1,30 @@
+name: Zoltar upload
+
+on:
+  workflow_dispatch #:
+  # schedule:
+  #  - cron: "0 7 * * *"
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+      with:
+        submodules: recursive
```

Comment:
Do we need to pull the validations submodule for this?

Reply:
Yes as it is at the moment (simple adaptation from the US hub). The process of getting the csv files into a zoltar-ready json relies on the `quantile_io` and `covid19` modules in the validation routine. It would definitely be good to make this simpler in the long term though as it's quite a hassle and not very easy to set up for e.g. a new hub. (Though some of this is probably my lack of python experience as the main hurdle was getting different file structures to work together)

Reply:
Ok :+1:, could you please add this as a comment to avoid any future mistake (since we might not always dig through the history of PRs)?

Reply:
Good idea, thanks!